bridge-cli/src/main.rs

```diff
@@ -53,34 +56,26 @@ impl CliConfig {
             eth_connector_account_id: self
                 .eth_connector_account_id
                 .or(other.eth_connector_account_id),
+            config_file: self.config_file.or(other.config_file),
         }
     }
-}
-
-#[derive(Subcommand, Debug)]
-enum SubCommand {
-    Nep141Connector {
-        #[clap(subcommand)]
-        cmd: Nep141ConnectorSubCommand,
-    },
-    EthConnector {
-        #[clap(subcommand)]
-        cmd: EthConnectorSubCommand,
-    },
-}
-
-#[derive(ValueEnum, Clone, Debug)]
-enum Network {
-    Mainnet,
-    Testnet,
-}
 
-#[derive(Parser, Debug)]
-#[clap(version)]
-struct Arguments {
-    network: Network,
-    #[command(subcommand)]
-    cmd: SubCommand,
+    fn empty() -> Self {
```

Comment:
You can use [default](https://doc.rust-lang.org/std/default/trait.Default.html) instead this

Reply:
Done